about.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Purple coders Festival</title>
+    <link rel="icon" type="image/x-icon" href="images/favicon.jpg" />
+    <link rel="stylesheet" href="./style.css" />
+    <script
+      src="https://kit.fontawesome.com/eb57cf8ccf.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+<!-- ==========HEADER SECTION=================== -->
+    <header>
+      <div class="page-info">
+        <i class="header-icons fa fa-facebook-f"></i>
+        <i class="fa fa-twitter"></i>
+        <span class="info-label">English</span>
+        <span class="info-label">My Page</span>
+        <span class="info-label">Logout</span>
+      </div>
+      <div class="nav-bar">
+        <a href="./index.html"
+          ><img
+            src= "./images/purpleLogoBg.png"
+            alt="Purple coders logo"
+            class="nav-logo"
+        /></a>
+        <nav>
+          <div class="nav-links" id="navLinks">
+            <i class="fas fa-times" id="closeIcon"></i>
+            <ul class="nav-items" id="navItems">
+              <li><a href="about.html">About</a></li>
+              <li><a href="#schedule">Schedule</a></li>
+              <li><a href="#community">Community</a></li>
+              <li><a href="#sponsor">Sponsor</a></li>
+              <li><a href="#blog">Blog</a></li>
+            </ul>
```

Comment:
- [ ] Please add a link to the home page on the about page so users can easily navigate your site.

Reply:
- [ ] This issue is not yet resolved :x: 


- [ ] According to the previous reviewer, you should add the `home` link in the about page so that users can navigate to the home page from the about page. According to [these lines of codes in the index.htm file](https://github.com/Kel-nana/Purple-Coders---HTML-CSS-JavaScript-capstone-project/blob/24c7fbf53ac1721a0af63c418b76dce65f27c6ac/about.html#L26-L31), you've made your logo navigatable, and this is totally fine, however, in you mobile menu on the `about page`, you should add the `home` to the list of items so that users can navigate to the `home` page from the `about page`. Currently one can not navigate back to the home page from the `about page` as this is not a good user experience on a mobile device. please fix this. see the image below. I hope this helps

*add the home to the nav items*
![Screenshot from 2023-02-16 17-59-37](https://user-images.githubusercontent.com/59397159/219435914-7cbcfee7-ba65-405c-b556-cc0ec2586521.png)